ee/fleetd-chrome/src/tables/network_interfaces.ts

```diff
@@ -5,21 +5,17 @@ export default class TableNetworkInterfaces extends Table {
   columns = ["mac", "ipv4", "ipv6"];
 
   async generate() {
-    let ipv4, ipv6, mac;
-    try {
-      // @ts-expect-error @types/chrome doesn't yet have the getNetworkDetails Promise API.
-      const networkDetails = (await chrome.enterprise.networkingAttributes.getNetworkDetails()) as chrome.enterprise.networkingAttributes.NetworkDetails;
-      ipv4 = networkDetails.ipv4;
-      ipv6 = networkDetails.ipv6;
-      mac = networkDetails.macAddress;
-    } catch (err) {
-      console.warn(`get network details: ${err}`);
-    }
+    // @ts-expect-error @types/chrome doesn't yet have the getNetworkDetails Promise API.
+    const networkDetails = (await chrome.enterprise.networkingAttributes.getNetworkDetails()) as chrome.enterprise.networkingAttributes.NetworkDetails;
```

Comment:
Is the idea that an error in the `getNetworkingDetails` call just bubbles up? 

Perhaps it would be helpful to still catch the error here and throw a new one that wraps the error with some additional context? I believe we do that in the some of the orbit extension tables.

Reply:
@gillespi314 that's a great point. @jacobshandling volunteered to build a error handling solution that has errors bubble up for the table and for fleet. I'd love to check out what he creates, if not done by end of sprint, we should remove the table level error handling so we can have the fleet app error handling.

cc: @lucasmrod @sharon-fdm

Reply:
Curious to know what kind of additional context we can provide.

Reply:
Following up about this, reiterating that we've decided for Chrome tables with multiple API calls, we're catching and collecting any errors, `console.log`-ing them, then returning all successfully fetched results as column values. The error handling solution above is a good idea for future iterations on this system.